### PR TITLE
Updating omnibus dependencies to take advantage of new msys2 toolchain

### PR DESCRIPTION
**Pull request by tyler-ball**

### CODE REVIEW

File: kitchen-tests/cookbooks/base/Berksfile.lock [hunk 1]
```diff
@@ -0,0 +1,64 @@
+DEPENDENCIES
```
Comment (ksubrama) [inline kitchen-tests/cookbooks/base/Berksfile.lock]: We shouldn't be adding these right?

File: kitchen-tests/cookbooks/webapp/Berksfile.lock [hunk 2]
```diff
@@ -0,0 +1,68 @@
+DEPENDENCIES
```
Comment (ksubrama) [inline kitchen-tests/cookbooks/webapp/Berksfile.lock]: And this one

**Comment by tyler-ball** [comment]
@ksubrama I removed the `Berksfile.lock` files I added, since we don't lock anything that isn't going through CI

**Comment by ksubrama** [comment]
:+1:

**Comment by ksubrama** [comment]
The travis failure is expected - it's warning us of a vulnerability in nokogiri...  and we are working around that for now.

**Comment by danielsdeleo** [comment]
👍 but let's not let the nokogiri thing fester

**Comment by mwrock** [comment]
:+1: agreed @danielsdeleo I have a nokogiri branch in the wings but like the idea of getting things green before adding that in.

**Pull request merged**